From 452b9127410427837428e75062cc9fa17633d974 Mon Sep 17 00:00:00 2001
From: Balint Reczey <balint@balintreczey.hu>
Date: Sun, 20 Sep 2015 19:24:23 +0200
Subject: [PATCH 1/2] Add hardened1-linux-<cpu> ports support

Those ports ar based on simple <cpu> ports with a set of defaults
changed to provide better security.
---
 ostable      | 1 +
 triplettable | 1 +
 2 files changed, 2 insertions(+)

diff --git a/ostable b/ostable
index 10e0d3a..678196a 100644
--- a/ostable
+++ b/ostable
@@ -23,6 +23,7 @@ gnuabi64-linux		linux-gnuabi64		linux[^-]*-gnuabi64
 gnuspe-linux		linux-gnuspe		linux[^-]*-gnuspe
 gnux32-linux		linux-gnux32		linux[^-]*-gnux32
 gnu-linux		linux-gnu		linux[^-]*(-gnu.*)?
+gnuhardened1-linux	linux-gnuhardened1	linux[^-]*(-gnuhardened1.*)?
 gnueabihf-kfreebsd	kfreebsd-gnueabihf	kfreebsd[^-]*-gnueabihf
 gnu-kfreebsd		kfreebsd-gnu		kfreebsd[^-]*(-gnu.*)?
 gnu-knetbsd		knetbsd-gnu		knetbsd[^-]*(-gnu.*)?
diff --git a/triplettable b/triplettable
index 568a6b9..a2bd1e5 100644
--- a/triplettable
+++ b/triplettable
@@ -16,6 +16,7 @@ gnuabi64-linux-mips64	mips64
 gnuspe-linux-powerpc	powerpcspe
 gnux32-linux-amd64	x32
 gnu-linux-<cpu>		<cpu>
+gnuhardened1-linux-<cpu>	hardened1-linux-<cpu>
 gnueabihf-kfreebsd-arm	kfreebsd-armhf
 gnu-kfreebsd-<cpu>	kfreebsd-<cpu>
 gnu-knetbsd-<cpu>	knetbsd-<cpu>
-- 
2.1.4

